nested/track.py

```diff
@@ -18,15 +16,59 @@ def __init__(self, indices, params):
             csvreader = csv.reader(ins)
             for row in csvreader:
                 self.tags.append(row[0])
-                self.users.append(row[1])
-                self.dates.append(row[2])
+                self.dates.append(row[1])
 
     def partition(self, partition_index, total_partitions):
         return self
 
     def size(self):
         return 1
 
+class TermQueryParamSource(QueryParamSource):
+
+    def params(self):
+        result = {
+            "body": {
+                "query": {
+                    "match": {
+                        "tag": "%s" % random.choice(self.tags)
+                    }
+                },
+                "sort": [
+                    {
+                        "answers.date": {
+                            "mode" :  "max",
+                            "order": "desc",
+                            "nested_path": "answers"
+                        }
+                    }
+                ]
+            },
+            "index": None,
+            "type": None,
+            "use_request_cache": False
```

Comment:
Here you can do: `"use_request_cache": self._params["use_request_cache"]`. This makes it better aligned with Rally core.